atlasdb-config/src/test/java/com/palantir/atlasdb/factory/TransactionManagersTest.java

```diff
@@ -612,15 +612,6 @@ private TransactionManagers withLockImmutableTsOnReadOnlyTransaction(boolean opt
                 .build();
     }
 
-    @Test
-    public void metricsAreReportedExactlyOnceWhenUsingLocalService() throws IOException {
```

Comment:
You've removed two tests, but added none - are there any tests we can add to have confidence that this fix is correct?

Reply:
I see you've said
>Existing tests should suffice for correctness.
Removal of metrics is verified by removing the test involved

but I'm not 100% on the second point - removing the tests just removes the check that there are metrics, but surely it doesn't actively check that _no_ metrics are registered? That's what we want here, right?

Reply:
Fair enough, yep I'll replace that with a check that they are not registered.